common/model/events.js

```diff
@@ -107,10 +107,8 @@ export type Event = {|
   // not sure if it should be in the model, a question for Silver
   bookingType: ?string,
   schedule?: Array<Event>,
-  eventInfo?: {|
-    eventbriteId?: string,
-    isCompletelySoldOut?: boolean
-  |}
+  eventbriteId?: string,
```

Comment:
Looks like this flattening is what's breaking in prismic-parsers.

Reply:
Yea we now have 2 event models.
Annoying, but too much for this PR to fix.